lib/host.py

```diff
@@ -201,13 +201,13 @@ def execute_script(self, script_contents, shebang='sh', simple_output=True):
             script.write('#!/usr/bin/env ' + shebang + '\n')
             script.write(script_contents)
             script.flush()
-            self.scp(script.name, script.name)
+            self.scp(script.name, "/tmp/" + script.name.split('/')[-1])
```

Comment:
better use `os.path.basneame`

Reply:
thanks, made the change.

Reply:
Ping @ydirson

Reply:
I like this fixture :)

Reply:
Straight up borrowed from https://github.com/xcp-ng/xcp-ng-tests/pull/324 😁
It could be great to have this kind of generic fixtures available for all tests 👀 (eventually platform agnostic unlike the one I suggested)

Reply:
Thanks, please see the latest update if it works.

Reply:
it's all good for me